src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1810,6 +1827,26 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
         return $bSaveSuccess;
     }
 
+    private function filterUnchangedFieldsFromPostData(array $postFieldsLanguageConvertedWithAccessRights, string $tableName, string $id): array
+    {
+        $escapedTableName = $this->databaseConnection->quoteIdentifier($tableName);
+        $dataBeforeUpdateQuery = 'SELECT * FROM '.$escapedTableName.' WHERE `id` = :id';
+        $stmt = $this->databaseConnection->prepare($dataBeforeUpdateQuery);
+        $stmt->bindValue('id', $id);
+        $stmt->execute();
+        $dataBeforeUpdateQueryResult =  $stmt->fetchAll();
+        $dataBeforeUpdate = $dataBeforeUpdateQueryResult[0];
+        $filteredSetFields = [];
+        foreach ($postFieldsLanguageConvertedWithAccessRights as $postKey => $postValue) {
```

Comment:
If this is "simply" a filter method then the fact that this data was post data before is here not relevant. And "post" could be remove from any name here.

Reply:
done